src/main/java/app/controller/LabelController.java

```diff
@@ -26,11 +26,12 @@
 public class LabelController {
     private final LabelService labelService;
 
-    @PostMapping
+    @PostMapping("/{taskId}")
     @ResponseStatus(HttpStatus.CREATED)
     @Operation(summary = "Label creation", description = "Create and save a new label")
-    public LabelResponseDto createLabel(@Valid @RequestBody LabelRequestDto dto) {
-        return labelService.create(dto);
+    public LabelResponseDto createLabel(@Valid @RequestBody LabelRequestDto dto,
+                                        @PathVariable Long taskId) {
+        return labelService.create(dto, taskId);
     }
 
     @GetMapping("/by_project/{projectId}")
```

Comment:
Here too. Let's replace PathVariable with RequestParam and remove this.
```suggestion
    @GetMapping
```